boards/up/up_squared_pro_7000/doc/up_squared_pro_7000.rst

```diff
@@ -66,13 +66,13 @@ connection setup.
 Booting the UP Squared Pro 7000 Board using UEFI
 ================================================
```

Comment:
You need to rename `{up_squared_pro_7000,index}.rst`

Reply:
> You need to rename `{up_squared_pro_7000,index}.rst`

why ? it is in different directory now `boards\up` and the pattern change is at `boards\intel\`

Reply:
Yes, right, anyway would make sense to rename to make it consistent, all other docs are named as `index.rst`.